atlas/templates/Makefile.common.gotmpl

```diff
@@ -78,7 +78,9 @@ push-atlas: docker
 ifndef IMAGE_REGISTRY
 	@(echo "Please set IMAGE_REGISTRY variable in Makefile.vars to use push command"; exit 1)
 else
-	@docker push $(SERVER_IMAGE):$(IMAGE_VERSION)
+	# Verify no changes locally prior to pushing image
+	git diff --exit-code
```

Comment:
This might not work as expected, because the flag file could be stale. The way this makefile was implemented doesn't check for a stale docker build context like basic `docker build` does.

If this section is removed, docker will ensure the image is based on the current context, which in turn will make the output of `git diff --exit-code` useful for matching the image to the commit.
https://github.com/infobloxopen/atlas-cli/blob/f08e9012188f16811a8015376994f7af4fa54a03/atlas/templates/Makefile.common.gotmpl#L70-L75